tool.gpt

```diff
@@ -40,6 +40,15 @@ tools: service
 
 #!http://service.daemon.gptscript.local/click
 
+---
+name: filter
+description: filter the page to get elements based on specific id, html tag, or class.
+args: filter: (required) the class (eg. '.foo') or id (eg. '#foo') of an object.
+args: tabID: (required) The ID of the tab. If unspecified, a new tab will be created.
```

Comment:
How so? The filter method can't load a URL, so it has to be an existing tab/page.

Reply:
Oh, I didn't think about that lol. I just assumed you meant it to be optional because of the second sentence. If it is required, then the second sentence should be removed here instead.

Reply:
Should be resolved now.